src/fides/api/models/privacy_experience.py

```diff
@@ -219,260 +566,41 @@ class PrivacyExperience(Base):
-            db=db, region=region.value
-        )
-
-        privacy_center_notices: Query = get_privacy_notices_by_region_and_component(
-            db,
-            [region.value, region_country(region.value)],
-            ComponentType.privacy_center,
-        )
-        overlay_notices: Query = get_privacy_notices_by_region_and_component(
-            db, [region.value, region_country(region.value)], ComponentType.overlay
-        )
-
-        # See if we need to create a Privacy Center Experience for the Privacy Center Notices
-        if new_experience_needed(
-            existing_experience=privacy_center_experience,
-            related_notices=privacy_center_notices,
-        ):
-            privacy_center_experience = (
-                PrivacyExperience.create_default_experience_for_region(
-                    db, region, ComponentType.privacy_center
-                )
-            )
-            added_experiences.append(privacy_center_experience)
-
-        # See if we need to create a new overlay Experience for the overlay Notices.
-        if new_experience_needed(
-            existing_experience=overlay_experience, related_notices=overlay_notices
-        ):
-            overlay_experience = PrivacyExperience.create_default_experience_for_region(
-                db, region, ComponentType.overlay
-            )
-            added_experiences.append(overlay_experience)
-
-    return added_experiences
-
-
-def remove_config_from_matched_experiences(
-    db: Session,
-    experience_config: PrivacyExperienceConfig,
-    regions_to_unlink: List[PrivacyNoticeRegion],
-) -> List[PrivacyNoticeRegion]:
-    """Remove the config from linked PrivacyExperiences"""
-    if not regions_to_unlink:
-        return []
-
-    experiences_to_unlink: List[PrivacyExperience] = experience_config.experiences.filter(  # type: ignore[call-arg]
-        PrivacyExperience.region.in_(regions_to_unlink)
-    ).all()
-
-    for experience in experiences_to_unlink:
-        if experience_config.is_default:
-            experience.unlink_experience_config(db)
-        else:
-            experience.link_default_experience_config(db)
-    return [experience.region for experience in experiences_to_unlink]  # type: ignore[misc]
 
 
 def upsert_privacy_experiences_after_config_update(
     db: Session,
     experience_config: PrivacyExperienceConfig,
     regions: List[PrivacyNoticeRegion],
-) -> Tuple[List[PrivacyNoticeRegion], List[PrivacyNoticeRegion]]:
+) -> List[PrivacyNoticeRegion]:
     """
-    Keeps Privacy Experiences in sync with ExperienceConfig changes.
-    Create or update PrivacyExperiences for the regions we're attempting to link to the
-    ExperienceConfig.
+    Links regions to a PrivacyExperienceConfig via the PrivacyExperience table.
 
-    Assumes that components on the ExperienceConfig do not change after they're updated.
+    PrivacyExperiences are queried directly for the end-user in a specific region and pull
```

Comment:
nit: i may just make a note here in the docstring that this can potentially add or remove `PrivacyExperience` records, to make it clear what "linking" implies!